pkg/kubelet/kubelet_node_status.go

```diff
@@ -515,6 +517,8 @@ func (kl *Kubelet) setNodeStatusMachineInfo(node *v1.Node) {
 		}
 		node.Status.Allocatable[k] = value
 	}
+
+	node.Status.Allocatable[v1.ResourceNvidiaGPU] = *resource.NewQuantity(int64(kl.nvidiaGPUManager.AvailableGPUs()), resource.DecimalSI)
```

Comment:
From the code I see ["Allocatable represents the resources of a node that are available for scheduling."](https://github.com/kubernetes/kubernetes/blob/master/pkg/api/types.go#L2484), so I think it is OK to put it here. Could you let me know why allocatable is "```Capacity-Reserved```" and where I put the unallocated GPUs is more appropriate?